### DISCUSSION TITLE
Fix the flux listing user panel and owner picker at signup

### DISCUSSION THREAD

**Discussion by followeuphoria2** [inspireui/support#5453] [New Feature/Ideas Requests]
A very nice man from inspire ui gave me a solution which I really appreciate for adding the vendor option at signup but unless it’s added to the mstore plugin and flux listing source code I’ll have to read it every update which will be very inconvenient. Please add this to the new release. So that our vendors don’t have to sign up as customers this is the main feature for using the app 

 In addition it is missing the user panel https://drive.google.com/file/d/1qN2J5Xy3LT7m1iC6bVBhc1aHiVAUQle-/view?usp=drivesdk user panel - please make sure that the user panel is in the app . thats  the main function. even if you just iframe it or something its best native but something is better than nothing for now the app is just screens with out the intended listeo functionality  

2. if the user picks apple, fb, or SMS <<< there is no option to pick a user type like owner it just directly signs up. whats the solution to that. only manual  email has the option to pick very confusing for the user. 

Please add these to the next release not in 6 months it’s very important I’ve waited a year already .. 

The .dart files are already available please add them to the source code 

thank you very much

**Comment by followeuphoria2** [comment]
![2E11F47E-FB66-4D99-BA00-8AB6DFE79CA6](https://user-images.githubusercontent.com/64839848/146724623-d8164a32-13c2-4e61-8656-fa7c6581f77f.jpeg)
![7A49AC15-FE58-4816-A507-EB399E0577D2](https://user-images.githubusercontent.com/64839848/146724630-78ea097a-436f-4441-b6da-7a3c1572f618.jpeg)

**Comment by ab-ashagrie** [comment]
I also vote for this feature to be added to your update queue. It is a very important feature

**Comment by followeuphoria2** [comment]
In the meantime I just replaces the image on the sheep and the words ready to travel with a picture that says dashboard. It words but there is a problem with the double login on the iframe used by the owner dashboard 

this feature is the only way the template will be functional. Please share the link
